openapi.go

```diff
@@ -258,7 +270,7 @@ func RegisterOpenAPIOperation[T, B any](openapi *OpenAPI, route Route[T, B]) (*o
 	for _, params := range route.Operation.Parameters {
 		if params.Value.In == "path" {
 			if !strings.Contains(route.Path, "{"+params.Value.Name) {
-				return nil, fmt.Errorf("path parameter '%s' is not declared in the path", params.Value.Name)
+				panic(fmt.Errorf("path parameter '%s' is not declared in the path", params.Value.Name))
 			}
```

Comment:
The tests were previously passing because of a panic later in the code but it wasn't intentional, here it is, as it should.